astroid/tests/unittest_brain_numpy_core_fromnumeric.py

```diff
@@ -0,0 +1,82 @@
+# -*- encoding=utf-8 -*-
+# Copyright (c) 2017-2018 hippo91 <guillaume.peillex@gmail.com>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/master/COPYING.LESSER
+import unittest
+import contextlib
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+class SubTestWrapper(unittest.TestCase):
+    """
+    A class for supporting all unittest version wether or not subTest is available
+    """
+
+    def subTest(self, msg=None, **params):
+        try:
+            # For python versions above 3.5 this should be ok
```

Comment:
❓ Isn't `subTest` available from 3.4? If so, we could use it today without the need of `subtestMock`.

Reply:
Well done @PCManticore . You are totally right this mock is useless now.
I deleted it.